fastlane/platforms/android.rb

```diff
@@ -63,7 +63,7 @@
   lane :matchesque do
     match_dir = clone_match
 
-    # don't forget – lanes run inside of ./fastlane
+    # don't forget --- lanes run inside of ./fastlane
```

Comment:
Uh

Reply:
I like my utf-8 chars. Why shouldn't I use them in comments?

Reply:
You can use them in comments, but the file should probably have UTF-8 set in its variables if you do.  There's a Cop for this that says you should only use ASCII comments.

Reply:
Then we're going to need to add the magic comment to all the files. I guarantee you that I'll want to put in more Unicode over time. Blame the Mac keyboard.

Reply:
(So, like, `Rølvaag` is cool, but `—` or `–` is not, at least, according to RuboCop.)

Reply:
… how does that make any sense?

Reply:
Silly rubocop.